### PR TITLE
Create LIT6956MalkeaMadAlam.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a 'provisional' record according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2508.

### CODE REVIEW

File: new/LIT6956MalkeaMadAlam.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```
Comment (eu-genia) [inline new/LIT6956MalkeaMadAlam.xml:37]: maybe here somewhere the information on the length (33 stanzas, as far as we know) should be inserted

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6956MalkeaMadAlam.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <revisionDesc>
+            <change who="CH" when="2024-04-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div xml:lang="gez" type="edition">
+                <note>The following incipit is taken from <ref type="mss" corresp="EMML6251"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>ካልዕ፡ መልክዓ፡ መድኃኔ፡ ዓለም፡ መጋጊት፡ ፳፯፡
```
Comment (DenisNosnitsin1970) [inline new/LIT6956MalkeaMadAlam.xml:60]: መጋቢት?

File: new/LIT6956MalkeaMadAlam.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6956MalkeaMadAlam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Malkǝʾa Madḫane ʿAlam - ʾƎsaggǝd</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6956MalkeaMadAlam.xml:8]: Please shape the title according to other malke' records, cp. LIT1867Malkea or LIT6158MalkeaGalawdewos (the second one seems to be a more recent muster). Somewhere (in abstracts or `<title xml:lang="gez" xml:id="t2">`) the date 27 Maggabit can be mentioned, and encoded in the text below  as `<date/>`.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6956MalkeaMadAlam.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6956MalkeaMadAlam.xml:60]: This line is from EMML6251, but is not in BLorient13273. Probably I should omit the first line as it is not representative and not part of the actual work.

Comment (DenisNosnitsin1970) [reply new/LIT6956MalkeaMadAlam.xml:60]: The line should not necesserily be quoted, but the infor on 27 Maggabit can be kept. And according to the line, this is "another malke'", distinguished from the first one.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6956MalkeaMadAlam.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6956MalkeaMadAlam.xml:8]: Should this date be specified or is it enough to encode `<date>መጋቢት፡ ፳፯፡</date>`?

Comment (DenisNosnitsin1970) [reply new/LIT6956MalkeaMadAlam.xml]: It can be specified in abstract, or you can make t2 "Image to the Saviour of the World for 27 Maggabit"

**Review by eu-genia — APPROVED** [review]

**Comment by smaugustine** [comment]
Just two small notes:

The numeral ፪ is to mark the start of the second stanza

I also wonder whether it's worth including as a secondary title "መልክአ፡ መድኃኔ፡ ዓለም፡ እሰግድ፡" as it is given here (no. 14):

E96441E3-ECCC-4289-B443-670B9EC52BCA_1_201_a

Numberings like መልክአ፡ X ካልእ፡ and መልክአ፡ X ሣልሳይ፡ are unreliable, but these incipit based ones (e.g. also መልክአ፡ ጊዮርጊስ፡ ዘሰሌዳ፡ ሞገስ፡, መልክአ፡ ጊዮርጊስ፡ ዘፅንሰቱ፡) are guaranteed to at least be consistent, even if they are not always used.

File: new/LIT6956MalkeaMadAlam.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div xml:lang="gez" type="edition">
+                <note>The following incipit is taken from <ref type="mss" corresp="EMML6251"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>መልክዓ፡ መድኃኔ፡ ዓለም፡ <date>መጋቢት፡ ፳፯፡</date>
+                        <lg n="1" type="stanza" xml:id="s1">
+                            <l n="1">፩  እሰግድ፡ ለጽንሰትከ፡ ወለልደትከ፡ መድምም።</l>
```
Comment (DenisNosnitsin1970) [inline new/LIT6956MalkeaMadAlam.xml:65]: I do not think we need ፩  and ፪ in this case.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
I seems that a part of the hymn (1 stanza?) is present also in the Synaxarion.

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**